FlightStreamDeck.AddOn/MainWindow.xaml.cs

```diff
@@ -81,21 +84,26 @@ private async void Window_Loaded(object sender, RoutedEventArgs e)
 
         private async Task InitializeSimConnectAsync(SimConnectFlightConnector simConnect)
         {
-            myNotifyIcon.Icon = new Icon("Images/button@2x.ico");
-            while (true)
-            {
-                try
+            AsyncRetryPolicy _retryPolicy = Policy
+                .Handle<COMException>()
+                .WaitAndRetryAsync(3, retryAttempt =>
+                {
+                    TimeSpan timeToWait = TimeSpan.FromSeconds(Math.Pow(2, retryAttempt));
+                    return timeToWait;
+                }, (Exception ex, TimeSpan ts) => {
+                    logger.LogError(ex, $"Waiting {ts.TotalSeconds} seconds before retrying...");
+                }
+            );
+
+            await _retryPolicy
+                .ExecuteAsync(() =>
                 {
+                    myNotifyIcon.Icon = new Icon("Images/button@2x.ico");
```

Comment:
I thought this would work, and tried even disposing both the icon and the notify icon before resetting it.  The notification area still gets spammed with icons till you mouse over it and then it goes away.  I'm not really familiar with these icons, any idea how we can stop that from happening @nguyenquyhy ?

If you start up the client with these changes and don't have MSFS running, you'll see what I am talking about.